### DISCUSSION TITLE
File Recognition Glitch

### DISCUSSION THREAD

**Discussion by MikeyProHD** [Raytwo/ARCropolis#476] [Issues] [answered]
For some reason every time I boot up Dmash with mods, it keeps saying that the romfs arc method had been deprecated,  even though I have nothing in that folder...

**Comment by Raytwo** [comment, marked as answer]
Having the folder is enough to display the message, just remove it if it bothers you.

**Reply by MikeyProHD** [reply]
Ok! Thanks a lot
